server/load_mock_data.py

```diff
@@ -19,12 +21,30 @@ def load():
             password='password'
         )
     ]
+    # Add test data
+    test_projects = [
```

Comment:
This is my bad since I had it the other way before, but we should be using the functions we made to add things to the database, rather than manually adding them. E.g., make a `dict` object and then call `create_assignment`, etc.